### DISCUSSION TITLE
Enable public/unauthenticated access

### DISCUSSION THREAD

**Discussion by nickj-git** [mealie-recipes/mealie#3061] [Feature Request]
### First Check

- [X] I used the GitHub search to find a similar requests and didn't find it.
- [X] Checked the [tasks tagged](https://github.com/hay-kot/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

### Please provide a concise description of the problem that would be addressed by this feature.

Using Mealie on a tablet in the kitchen works nice, but the need to authenticate makes it much less user friendly, especially as it is a self hosted solution that I don't expose to the internet. It makes sense that unauthenticated user receive certain permissions so that any device at home can view and edit the meal plan without requiring to log in.

### Please provide a concise description of the feature that would resolve your issue.

Enable a level of permission to unauthenticated/public users, for example to view all recipes, view and edit the meal plan, etc. OR have the ability to create a "public" non-admin user.

### Please consider and list out some caveats or tradeoffs made in your design decision

If the permissions cannot be managed like described above, maybe you can create a public non-admin user instead that does not require to log in which may reduce the need to developer specific user access which I'd imagine is a lot more work.

### Additional Information

- [ ] If this is accepted I'm willing to submit a PR to provide this feature
- [ ] If this is accepted I'm willing to help maintain this feature
- [ ] I'm willing to sponsor/pay a developer to do this work

**Comment by michael-genson** [comment]
Is this what you're looking for?

https://nightly.mealie.io/documentation/getting-started/usage/permissions-and-public-access/

**Reply by nickj-git** [reply]
Thank you. Not entirely, the recipes are definitely public which is also needed in my use case,  but it would be great for a kitchen tablet without logging in to be able to view and edit the meal plan. I can't set that to public I believe?

**Reply by michael-genson** [reply]
Currently there's no way to view the meal planner without logging in, however you can set the token time to an arbitrarily large number (e.g. 10 years) and then you only need to log-in once.

More here: https://nightly.mealie.io/documentation/getting-started/installation/backend-config/

**Reply by nickj-git** [reply]
Thanks! I guess that should do the job too. I'll give that a go :)
